src/main/java/com/google/sps/servlets/getDailyQuiz.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.QueryResultList;
+import java.text.DateFormat;
+import java.util.Date;
+import com.google.appengine.api.datastore.Key;
+
+@WebServlet("/getQuizStatus")
+public class getDailyQuiz extends HttpServlet {
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        
+        Query query = new Query("Quiz");
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery pq = datastore.prepare(query);
+
+        Entity quiz = pq.asList(FetchOptions.Builder.withLimit(1)).get(0);
```

Comment:
If I understand this correctly, there is only one quiz for the entire website? There is not one quiz generated per user? So everyone just gets the same quiz? And, if the day has rolled over, we generate a new quiz for users to take instead? That seems fine if this is the case, just wanted to make sure I'm understanding this correctly. If this is the case though, we could simplify this logic and make fewer database calls by storing the quiz as a variable in the class itself and do something like:

If class.quiz == null:
--check database for existing quiz
--if no existing quiz
----make new quiz
----store in class.quiz
----store in database
--else if existing quiz in database
----store existing quiz in class.quiz
else if class.quiz
--return class.quiz